### PR TITLE
MAT-8750: Adding unlock on logout/timeout

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-8750](https://jira.cms.gov/browse/MAT-8750)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included
* [X] No extemporaneous files are included (i.e Complied files or testing results)
* [X] This PR is into the **correct branch**.
* [X] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/timeoutHandler/TimeoutHandler.tsx [hunk 1]
```diff
@@ -22,6 +26,13 @@ const TimeoutHandler = ({ timeLeft = 10000, warningTime = 5000 }) => {
   // check if component is mounted before memory leak
   const mounted = useRef(false);
   useEffect(() => {
+    const setRef = async () => {
+      //eslint-disable-next-line
+      measureServiceApiRef.current = await useMeasureServiceApi();
+      //eslint-disable-next-line
+      cqlLibraryServiceApiRef.current = await useCqlLibraryServiceApi();
+    };
+    setRef();
```
Comment (mcmcphillips) [inline src/components/timeoutHandler/TimeoutHandler.tsx]: I don' t think that we want to await the apis
I believe that this caused issues previously but can't recall

@RohitKandimalla

**Review by mcmcphillips — APPROVED** [review]
